tests/unit/AddEditBusCorp.ts

```diff
@@ -4,7 +4,7 @@ import Vuelidate from 'vuelidate'
 import { mount, Wrapper, createLocalVue } from '@vue/test-utils'
 import flushPromises from 'flush-promises'
 import { getVuexStore } from '@/store'
-import OrgPerson from '@/components/common/OrgPerson.vue'
+import AddEditBusCorp from '@/components/common/AddEditBusCorp.vue'
```

Comment:
GitHub diff is confused. This file is actually the new test file (cloned from OrgPerson). The next file below is the renamed one.